specs/subresourceintegrity/spec.markdown

```diff
@@ -318,7 +330,21 @@ resources accessed over a `file` scheme URL are unlikely to be
 eligible for integrity checks.
 {:.note}
 
+One should note that being a [privileged context][] (e.g. HTTPS) is not
+necessary for the use of integrity validation. Because resource integrity is
+only an application level security tool, and it does not change the security
+state of the user agent, a privileged context is unnecessary. However, if
+integrity is used in an [unprivileged context][] (e.g. HTTP), users should be
```

Comment:
"web developers" is preferred by the community I believe. "authors" is the formal term and used elsewhere in the specification it seems.